diff --git a/node_modules/@slack/interactive-messages/dist/adapter.js b/node_modules/@slack/interactive-messages/dist/adapter.js
index be6b81e..3f173bc 100644
--- a/node_modules/@slack/interactive-messages/dist/adapter.js
+++ b/node_modules/@slack/interactive-messages/dist/adapter.js
@@ -243,6 +243,9 @@ var SlackMessageAdapter = /** @class */ (function () {
     SlackMessageAdapter.prototype.requestListener = function () {
         return http_handler_1.createHTTPHandler(this);
     };
+    SlackMessageAdapter.prototype.all = function (callback) {
+        return this.registerCallback('*', callback);
+    };
     /* Interface for adding handlers */
     /* tslint:disable max-line-length */
     /**
